backend/oasst_backend/api/v1/auth.py

```diff
@@ -0,0 +1,49 @@
+import json
+from typing import Union
+
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.hkdf import HKDF
+from fastapi import APIRouter, Depends, Security
+from fastapi.security import APIKeyCookie
+from jose import jwe
+from oasst_backend.config import settings
+from pydantic import BaseModel
+
+router = APIRouter()
+
+oauth2_scheme = APIKeyCookie(name=settings.AUTH_COOKIE_NAME)
+
+
+class TokenData(BaseModel):
+    """
+    A minimal re-creation of the web's token type.  To be expanded later.
+    """
+
+    email: Union[str, None] = None
```

Comment:
I believe pydantic has email types, but depends if validation is an issue here

Reply:
Nice! switched to `EmailStr` and installed the required sub-package.